### PR TITLE
SacrificeOnce Trigger

### LINKED ISSUE #5674: Camellia, the Seedmiser
Needs "SacrificedOnce" trigger

Because the trigger cares about "You" sacrificing stuff, it should collect the LKI and trigger per Player.

Most sacrifice stuff already applies to a List, or is already in a for each player loop

### PR DESCRIPTION
**Pull request by Hanmac**
Some things i need to consider:

- [x] I really don't like the Destroy Effect that does Sacrifice instead, we should refactor them #5719
- ~~[ ] For Cost, i don't know, but i feel we need to handle them different for stuff that does "Sacrifice itself AND other", for this special case, maybe it shouldn't trigger there :(~~

Closes #5674

### CODE REVIEW

**Comment by tool4ever** [comment]
you need to add it to `registerActiveLTBTrigger`

**Comment by Hanmac** [comment]
> you need to add it to `registerActiveLTBTrigger`

@tool4ever feel free to add it

File: forge-game/src/main/java/forge/game/ability/effects/DestroyEffect.java [hunk 1]
```diff
@@ -71,12 +73,37 @@ public void resolve(SpellAbility sa) {
...
+
+        final boolean remDestroyed = sa.hasParam("RememberDestroyed");
+
+        SpellAbility cause = sa;
+        if (sa.isReplacementAbility()) {
+            cause = (SpellAbility) sa.getReplacingObject(AbilityKey.Cause);
+        }
+
+        // TODO remove Sacrifice inside Destroy
+        for (Card c : game.getAction().sacrifice(sacList, cause, true, params)) {
```
Comment (Hanmac) [inline forge-game/src/main/java/forge/game/ability/effects/DestroyEffect.java:101]: @tool4ever if we would remove the Destroy/Sacrifice thing, I could drop this extra piece of code

Comment (Hanmac) [reply forge-game/src/main/java/forge/game/ability/effects/DestroyEffect.java:101]: Waiting for #5744 to be merged first

**Comment by Hanmac** [comment]
@tool4ever i think this MR is ready?

**Review by tool4ever — APPROVED** [review]

**Pull request merged**